apps/api/src/main.ts

```diff
@@ -71,7 +71,7 @@ async function bootstrap() {
       `,
     )
     .setVersion('1.0.0')
-    .addApiKey({ type: 'apiKey', name: 'x-api-key', in: 'header' })
+    .addApiKey({ type: 'apiKey', name: 'x-api-key', in: 'header' }, 'apiKey')
```

Comment:
Thanks for your contribution.

Can you add same `.addApiKey({ type: 'apiKey', name: 'x-api-key', in: 'header' }, 'apiKey')` code to adminDocumentConfig as well in line 55?

It seems feedback creation API in admin API needs it.

Reply:
<img width="1694" alt="image" src="https://github.com/line/abc-user-feedback/assets/6156679/ba3ffbe3-e0f0-491f-9b67-f3089dc4a3e7">

Yes I will do it.:)